### PR TITLE
Add two SFR related plots

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
- M200 - SFR / Mstar for active galaxies
 - M200 - SFR / M200 for active galaxies

### CODE REVIEW

**Comment by JBorrow** [comment]
Initially looks okay to me. If you want your new plots to show up in the webpages you will need to edit `data_conversion/index.html`: https://github.com/JBorrow/xl-pipeline/blob/master/data_conversion/index.html. If you are happy with them not showing up as is I am happy to accept the changes.

**Comment by MatthieuSchaller** [comment]
Thanks. Will do.

**Comment by MatthieuSchaller** [comment]
Is that the correct way of doing it?

File: data_conversion/index.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <a href="#sfr">Star Formation Rates</a>
     <a href="#bh">Black Holes</a>
     <a href="#smhmr">Stellar Mass-Halo Mass Relation</a>
+    <a href="#sfr_halo">Star Formation Rates-Halo Mass Relation</a>
```
Comment (JBorrow) [inline data_conversion/index.html]: No need for plural in star formation rate(s) when putting it as the first one, the plural is referring to the number of plots. Perhaps we should have changed it to `Stellar Mass-Halo Mass Relations`...

File: data_conversion/index.html [hunk 2]
```diff
@@ -106,6 +107,7 @@ <h1>Black Holes</h1>
         <div class="plots">
             <img class="plot" src="stellar_mass_black_hole_mass_100.png" />
             <img class="plot" src="stellar_veldisp_black_hole_mass_10.png" />
+	    <img class="plot" src="halo_mass_black_hole_mass.png" />
```
Comment (JBorrow) [inline data_conversion/index.html]: Indentation at wrong level

File: data_conversion/index.html [hunk 3]
```diff
@@ -106,6 +107,7 @@ <h1>Black Holes</h1>
...
         </div>
         <p>
             Black-hole related quantities. Note that the stellar velocity
```
Comment (JBorrow) [inline data_conversion/index.html:113]: Please add comment describing `halo_mass_black_hole_mass.png`

File: data_conversion/index.html [hunk 4]
```diff
@@ -129,6 +131,19 @@ <h1>Stellar Mass-Halo Mass Relation</h1>
         </p>
     </div>
 
+    <div class="plotrow" id="sfr_halo">
```
Comment (JBorrow) [inline data_conversion/index.html:137]: Would it not be better to include this at the `Star Formation Rates` section?

Comment (MatthieuSchaller) [reply data_conversion/index.html:137]: I wanted to keep it separate as this is directly not observable and has no direct data comparison.

**Comment by MatthieuSchaller** [comment]
I have addressed your comments.

**Comment by JBorrow** [comment]
Thanks!

**Pull request merged**